easybuild/easyconfigs/x/XCfun/XCfun-20180122-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'MakeCp'
+
+name = 'XCfun'
+version = '20180122'
+commit = 'acd11bf'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://dftlibs.org/xcfun/'
+description = """ XCFun is a library of DFT exchange-correlation (XC) functionals.
+ It is based on automatic differentiation and can therefore generate arbitrary order
+ derivatives of these functionals. """
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/dftlibs/xcfun/archive/']
+sources = [{'download_filename': '%s.tar.gz' % commit, 'filename': SOURCE_TAR_GZ}]
+patches = ['%(name)s-%(version)s-shared_library-intel.patch']
+checksums = [
+    # acd11bfbdcc2d2fb9c100111545bd479e6e2c452.tar.gz
```

Comment:
@smoors Please also update this comment, should be renamed source tarball (`XCfun-20180122.tar.gz`)